k8s/commander.sh

```diff
@@ -34,9 +34,9 @@ function deploy {
 
     post-deploy "$@"
 
-    if command -v slack-cli > /dev/null; then
-        slack-cli -d "${SLACK_CHANNEL}" ":tada: Successfully deployed <${DOCKER_HUB}|full-${COMMIT_HASH}> to <https://${REGION_ENV}-${REGION}.sumo.mozit.cloud/|SUMO-${REGION_ENV} in ${REGION}>"
-    fi
+    #if command -v slack-cli > /dev/null; then
+    #    slack-cli -d "${SLACK_CHANNEL}" ":tada: Successfully deployed <${DOCKER_HUB}|full-${COMMIT_HASH}> to <https://${REGION_ENV}-${REGION}.sumo.mozit.cloud/|SUMO-${REGION_ENV} in ${REGION}>"
```

Comment:
Could we echo similar to this line (but without the weird slack formatting), just to make pasting it into #sumodev easier?